frontend/src/component/common/SegmentItem/SegmentItem.tsx

```diff
@@ -41,6 +41,9 @@ const StyledAccordion = styled(Accordion)(() => ({
 const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => ({
     fontSize: theme.typography.body2.fontSize,
     minHeight: 'unset',
+    ':focus-within': {
+        backgroundColor: 'inherit',
+    },
```

Comment:
Prevent background color from being set to a darker shade when focused. we don't set an explicit background color for it, so we'll set it to "inherit" on focus-within.